modules/apps/frontend-js/frontend-js-tooltip-support-web/src/main/resources/META-INF/resources/index.js

```diff
@@ -26,34 +26,33 @@ import React, {
 import reducer, {STATES} from './reducer';
 
 const ALIGN_POSITIONS = [
-	'top-right',
 	'top',
-	'top-left',
+	'top-right',
+	'right',
 	'bottom-right',
 	'bottom',
 	'bottom-left',
 	'left',
-	'right',
+	'top-left',
 ];
 
+const ALIGN_POSITIONS_MAP = {
+	bottom: POSITIONS.Bottom,
+	'bottom-left': POSITIONS.BottomLeft,
+	'bottom-right': POSITIONS.BottomRight,
+	left: POSITIONS.Left,
+	right: POSITIONS.Right,
+	top: POSITIONS.Top,
+	'top-left': POSITIONS.TopLeft,
+	'top-right': POSITIONS.TopRight,
+};
+
 const SELECTOR_TOOLTIP = '.tooltip[role="tooltip"]';
 const SELECTOR_TRIGGER = `
 	.lfr-portal-tooltip,
```

Comment:
Not sure if this is needed because for `.lfr-portal-tooltip` to actually work, the element needs to have either a `title` or `data-title` attribute, so that should already be covered by the new selector.

My main concern with this is that we now tooltip all the things whereas before we didn't... I'm wondering if it's possible that people won't like our tooltips and this might be too invasive... 🤔

Reply:
> Not sure if this is needed because for `.lfr-portal-tooltip` to actually work, the element needs to have either a `title` or `data-title` attribute, so that should already be covered by the new selector.

Yeah, we can safely remove this :+1: 

> My main concern with this is that we now tooltip all the things whereas before we didn't... I'm wondering if it's possible that people won't like our tooltips and this might be too invasive...

If there is a `title` attribute, browser will show default tooltip. We are only making them look nicer. If people don't find them nicer, we should ping our design team :slightly_smiling_face: 

I don't know if there are any browser-specific quirks or edge behaviors that prevent tooltips on some element, these might pop up unintentionally. In this case, if it's unacceptable to the product team, there is always an option of not setting the `title` attribute. We may have some opinionated tags or components that always set them. If those are a problem, we can extend their API to allow no tooltips.